packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1619,3 +1623,272 @@ Multiple IdP instances can be defined in a single Policy Action. This allows use
     }
   }
 ```
+
+## App Sign On Policy 
+
+<ApiLifecycle access="ie" /><br>
+
+> **Note:** This feature is only available as a part of the Okta Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
+
+The app sign-on policy determines the extra levels of authentication (if any) that must be performed before a specific Okta application can be invoked. It is always associated with an app via a Mapping. The Okta Identity Engine always evaluates both Okta sign-on policy and the sign-on policy for the app. The resulting user experience will be the union of both policies. App sign on policies have the type `ACCESS_POLICY`.
+
+> **Note:** You can have a maximum of 5000 app sign-on policies in an org.
+> There is a max limit of 100 rules allowed per policy.
+> When you create an app sign-on policy, you will automatically also create a default policy rule with the lowest priority of `99`.
+> The highest priority that an app sign on policy rule can be set is `0`.
+
+
+#### App Sign On Policy example
+
+```json
+    {
+        "type": "ACCESS_POLICY",
+        "name": "Web Cart App Sign On Policy",
+        "description": "Standard policy for Web Cart application"
+    }
+```
+
+### Policy conditions
+
+Policy conditions are not supported, conditions are applied at the rule level for this type of policies.
+
+### Policy Rules conditions
+
+The following conditions may be applied to the rules associated with an app sign on policy:
+
+* [People Condition](#people-condition-object)
```

Comment:
IDX has to doc `people` and `usertype` conditions

Reply:
People condition is already present in the doc. UserType condition will be added later.